python/Ganga/Runtime/GPIexport.py

```diff
@@ -18,6 +18,31 @@
 from Ganga.GPIDev.Base.Objects import GangaObject
 from inspect import isclass
 
+def _addToInterface(interface, name, _object):
+
+    if isType(_object, GangaObject):
+        setattr(interface, name, addProxy(_object))
+    elif isclass(_object) and issubclass(_object, GangaObject):
+        setattr(interface, name, getProxyClass(_object))
+    else:
+        setattr(interface, name, _object)
+
+def exportToInterface(myInterface, name, _object, doc_section, docstring=None):
```

Comment:
To access the list of jobs currently available I'd say Tasks as you want to do things such as get RegistrySlices which are useful for other tasks in core, otherwise you've got to make your own from the registry which is more complex than just calling the ```JobRegistrySliceProxy```.

The ```RegistrySlice``` is used heavily in monitoring (but I suspect it can be swapped out for a plan dictionary there or directly use the registry itself. It is also used/returned when performing a ```select``` operation and I'd say having a ```Task``` perform a ```jobs.select``` is quite a powerful thing to allow for job management.

Looking through the plugins gets difficult if you're just parsing ```"LocalFile"``` into ```LocalFile``` it's much easier to just look for ```"LocalFile"``` within the interface dictionary I would have thought otherwise you've got to iterate over the nested dictionaries in the Plugins to determine if the string can be parsed in this way.
Maybe a better solution would be a ```pluginLookUp``` function of some sort which does this but for now there are areas where I'm parsing the interface within the Proxy code to keep old behavior.